styles/_pages.scss

```
@@ -1060,7 +969,8 @@
 
 .page-docs-index {
   &::before {
-    background-image: url(../img/backgrounds/bg-docs.png);
+    background-image: url(../../img/backgrounds/docs-splash.svg);
```

If you move the file as I recommended above, update this path as well.

```suggestion
    background-image: url(../../assets/img/backgrounds/docs-splash.svg);
```